lib/containers/common.pm

```diff
@@ -96,6 +96,17 @@ sub install_docker_when_needed {
     assert_script_run('docker info');
 }
 
+sub install_buildah_when_needed {
+    my $host_os = shift;
+    my @pkgs    = qw(buildah);
+    if (script_run("which buildah") != 0) {
+        # We may run openSUSE with DISTRI=sle and opensuse doesn't have SUSEConnect
+        add_suseconnect_product('sle-module-containers') if ($host_os =~ 'sles' && is_sle('>=15'));
+        zypper_call "in @pkgs";
```

Comment:
maybe we can use specific package name instead of array, as there are not more packages to install, just as we do for `install_docker_when_needed`.

Reply:
Actually there is. see later i install `podman`. i think it make sense to add it here and remove the `install_podman_when_needed($host_distri);` wdys?

Reply:
Checkout the new code